calibration_node:
  ros__parameters:
    wheel_radius: 0.1115
    robot_baseline: 0.52
    ramp_gain: 0.05
    cmd_rate: 20
    encoder_rate: 30
    max_lin_speed: 2.0
    min_lin_speed: -2.0
    lin_speed_step: 0.5
    max_ang_speed: 5.0
    ang_steps: 8
    step_len: 4.0
    dead_man_button: True
    dead_man_index: 5
    dead_man_threshold: 0.7
    ramp_trigger_button: False
    ramp_trigger_index: 7
    calib_trigger_button: False
    calib_trigger_index: 7
    steady_state_window: 1.5
    steady_state_std_dev_threshold: 0.5
    n_uniform_samples: 25
    n_validation_samples: 25
    load_input_space_calib_data: False
    run_random_validation: True
    path_to_input_space_calib_data: "/home/robot/workspaces/marmotte_ws/src/doughnut_calib/calib_data/marmotte/input_space_data_mw.pkl"